jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainPatch.java

```diff
@@ -798,19 +801,22 @@ protected void setLodBottom(int lodBottom) {
 
     @Override
     public int collideWith(Collidable other, CollisionResults results) throws UnsupportedCollisionException {
-        if (refreshFlags != 0)
-            throw new IllegalStateException("Scene graph must be updated" +
-                                            " before checking collision");
-
-        if (other instanceof BoundingVolume)
-            if (!getWorldBound().intersects((BoundingVolume)other))
+        if ((refreshFlags & (RF_BOUND | RF_TRANSFORM)) != 0) {
+            logger.warning("Scene graph must be updated before checking collision");

```

Comment:
Hi guys, just 2 cents. Throughout the engine the log is used like this:
`logger.log(Level.WARNING, ...)` instead of `logger.warning(...)`. Maybe it's better to keep the same rule here too, just for a matter of style. What do you think?

Reply:
Good idea!

Reply:
I updated the file with this change